templates/graphdb-master.yaml

```diff
@@ -59,29 +74,41 @@ spec:
           volumeMounts:
             - name: graphdb-master-storage
               mountPath: /opt/graphdb/home
-            {{- if eq ($master_index | int64) 1 }}
-            - name: graphdb-backup
-              mountPath: /opt/graphdb/backups
-            {{- end }}
+          #TODO: The backup should be reworked to work with an array of repositories. The restore must get a
+          #repository name from the values file, but first we have to rework the restore procedure so the
+          #backups must be placed in proper folders according to the repository name
+{{/*            {{- if eq ($master_index | int64) 1 }}*/}}
+{{/*            - name: graphdb-backup*/}}
+{{/*              mountPath: /opt/graphdb/backups*/}}
+{{/*            {{- end }}*/}}
           resources: {{ $.Values.graphdb.masters.resources | toYaml | nindent 12 }}
           # Allow for GraphDB to start within 10*30 seconds before readiness & liveness probes interfere
           # TODO: Check what will happen if some pod dies
           startupProbe:
             httpGet:
               path: /repositories
               port: graphdb-m-{{ $master_index }}
+              httpHeaders:
+                - name: Authorization
+                  value: 'Basic {{ $authToken }}'
             failureThreshold: 30
             periodSeconds: 10
           readinessProbe:
             httpGet:
               path: /repositories
               port: graphdb-m-{{ $master_index }}
+              httpHeaders:
+              - name: Authorization
+                value: 'Basic {{ $authToken }}'
             initialDelaySeconds: 5
             periodSeconds: 10
           livenessProbe:
```

Comment:
Consider the tips from https://srcco.de/posts/kubernetes-liveness-probes-are-dangerous.html
Maybe at least increase the failure threshold count

Reply:
Talked with Pavel, he advised to use some protocol endpoint, not the repositories endpoint as it might cause issues if there are no repositories or the active location is changed